package/network/utils/wireguard-tools/Makefile

```diff
@@ -36,7 +36,9 @@ define Package/wireguard-tools
   URL:=https://www.wireguard.com
   MAINTAINER:=Jason A. Donenfeld <Jason@zx2c4.com>
   TITLE:=WireGuard userspace control program (wg)
-  DEPENDS:=+@BUSYBOX_CONFIG_IP +@BUSYBOX_CONFIG_FEATURE_IP_LINK
+  DEPENDS:=+@BUSYBOX_CONFIG_IP +@BUSYBOX_CONFIG_FEATURE_IP_LINK \
+	  +LINUX_5_4:kmod-wireguard-oot \
+	  +!LINUX_5_4:kmod-wireguard
```

Comment:
Either separate lines for everything or fill up lines until max. 75 chars.

Reply:
I will rework wireguard into a virtual kernel package and back out this change.